client/blocks/signup-form/index.jsx

```diff
@@ -783,43 +783,36 @@ class SignupForm extends Component {
 		recordTracksEvent( 'calypso_signup_tos_link_click' );
 	};
 
-	getTermsOfServiceUrl() {
-		return localizeUrl( 'https://wordpress.com/tos/' );
-	}
+	handleOnClickPrivacy = () => {
```

Comment:
Naming nit: `onClick` usually points to a function that will be executed on click. `handleClick` is usually that function. `handleOnClick` is a mix of both that I don't think makes much sense. Perhaps we should change this to just `handlePrivacyClick`?

Reply:
Sure, I'll change it. The convention was copied from the function above (`handleOnClickTos`) but then I'll update both of them.

Reply:
Thanks for that!

Reply:
Function names updated in https://github.com/Automattic/wp-calypso/pull/66220/commits/b0bb2d9b6e39bf15981db9cdc201db9417fd07ce